### PR TITLE
fix build

### PR DESCRIPTION
**Pull request by adamralph**

### CODE REVIEW

File: build.csx [hunk 1]
```diff
@@ -89,3 +96,36 @@ public static void Cmd(string fileName, string args)
         }
     }
 }
+
+public static string ReadCmd(string fileName, string args)
```
Comment (adamralph) [inline build.csx:100]: FYI I'm going to wrap both this and `Cmd()` into a NuGet package some time soon. Right now I'm repeating it in a few places.

File: tests/ConfigR.Tests.Acceptance.Roslyn.CSharp/Support/ConfigFile.cs [hunk 2]
```diff
@@ -27,7 +27,7 @@ public static IDisposable Create(string contents, string path)
         public static string GetDefaultPath()
         {
             AppDomain.CurrentDomain.SetData("APP_CONFIG_FILE", "Test.exe.config");
-            return Path.ChangeExtension(AppDomain.CurrentDomain.SetupInformation.ConfigurationFile, "csx");
+            return AppDomain.CurrentDomain.SetupInformation.ConfigurationFile;
```
Comment (adamralph) [inline tests/ConfigR.Tests.Acceptance.Roslyn.CSharp/Support/ConfigFile.cs:30]: This was a mistake. The app domain configuration file should never have been set to a csx file. This was causing https://github.com/config-r/config-r/blob/c66205b8d86881d49108e64e99b45df1f611902a/tests/ConfigR.Tests.Acceptance.Roslyn.CSharp/SearchPathsFeature.cs#L67 to fail. The current version of `HttpClient` was trying to read from the csx file and it was throwing an exception saying that it is malformed. Earlier versions of `HttpClient` must have been more forgiving.

**Review by adamralph — COMMENTED** [review]
FYI `build.cmd` is taken from my "standard" version over at https://gist.github.com/adamralph/c86eaf694c45ec4f8b6748eb3f13221d

**Comment by filipw** [comment]
thank you!
I have a branch where I'm moving this to the new dotnet project system, then the msbuild look up magic can go away and we could have a single build script for all platforms.

**Comment by adamralph** [comment]
👍 indeed, vswhere is only needed for projects which still use the old csproj. Still, I'm going to wrap the `Cmd()` stuff in a NuGet package since that's still needed for various things.

**Pull request merged**